surface/model_openapiv3.go

```diff
@@ -39,11 +40,12 @@ func newOpenAPI3Builder() *OpenAPI3Builder {
 	return &OpenAPI3Builder{model: &Model{}}
 }
 
-func (b *OpenAPI3Builder) buildModel(document *openapiv3.Document) (*Model, error) {
+func (b *OpenAPI3Builder) buildModel(document *openapiv3.Document, sourceName string) (*Model, error) {
```

Comment:
I notice that we're only adding symbolic references for OpenAPI v3. Perhaps we should note that explicitly.

Reply:
Sure, I can add a comment. Where is the best place to add the comment? I believe inside surface.proto right next to the `symbolicReferences` field it would be likely to get spotted.